Source/WebCore/workers/service/server/SWServerRegistration.cpp

```diff
@@ -423,4 +424,24 @@ std::optional<ExceptionData> SWServerRegistration::setNavigationPreloadHeaderVal
     return { };
 }
 
+void SWServerRegistration::addCookieChangeSubscriptions(Vector<CookieChangeSubscription>&& subscriptions)
+{
+    for (auto& subscription : subscriptions)
```

Comment:
You don't need this loop:
`m_cookieChangeSubscriptions.add(subscriptions.begin(), subscriptions.end());`